.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - 'etc/**'
+      - '**.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'etc/**'
+      - '**.md'
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  style-check:
+    name: Libraries Style Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        rust: [1.62.0]
```

Comment:
This is newer than the main repo's toolchain (`nightly-2022-01-06`, 1.59) but it's ok to me. CC @ShiKaiWi

Reply:
It's OK.